packages/local_auth/local_auth/README.md

```diff
@@ -1,148 +1,188 @@
 # local_auth
 
+<?code-excerpt path-base="excerpts/packages/local_auth_example"?>
+
 This Flutter plugin provides means to perform local, on-device authentication of
 the user.
 
-This means referring to biometric authentication on iOS (Touch ID or lock code)
-and the fingerprint APIs on Android (introduced in Android 6.0).
+On supported devices, this includes authentication with biometrics such as
+fingerprint of facial recognition.
 
 |             | Android   | iOS  |
 |-------------|-----------|------|
 | **Support** | SDK 16+\* | 9.0+ |
 
 ## Usage
 
-Import the relevant file:
-
-```dart
-import 'package:local_auth/local_auth.dart';
-```
+### Device Capabilities
 
-To check whether there is local authentication available on this device or not, call canCheckBiometrics:
+To check whether there is local authentication available on this device or not,
+call `canCheckBiometrics` (if you need biometrics support) and/or
+`isDeviceSupported()` (if you just need some device-level authentication):
 
+<?code-excerpt "readme_excerpts.dart (CanCheck)"?>
 ```dart
-bool canCheckBiometrics =
-    await localAuth.canCheckBiometrics;
+import 'package:local_auth/local_auth.dart';
+// ···
+  final LocalAuthentication auth = LocalAuthentication();
+  // ···
+    final bool canAuthenticateWithBiometrics = await auth.canCheckBiometrics;
+    final bool canAuthenticate =
+        canAuthenticateWithBiometrics || await auth.isDeviceSupported();
 ```
 
 Currently the following biometric types are implemented:
 
 - BiometricType.face
 - BiometricType.fingerprint
+- BiometricType.weak
+- BiometricType.strong
+
+### Enrolled Biometrics
+
+`canCheckBiometrics` only indicates whether hardware support is available, not
+whether the device has any biometrics enrolled. To get a list of enrolled
+biometrics, call `getAvailableBiometrics()`.
```

Comment:
`canCheckBiometrics` and `getAvailableBiometrics` sounds a bit unclear (that may be the reason we add so many comments to clarify it?)
Maybe `hasAnyBiometricsHardware` and `getEnrolledBiometrics`?